docs/reference/logging/fluentd.md

```diff
@@ -0,0 +1,86 @@
+# Fluentd logging driver
+
+The `fluentd` logging driver sends container logs to the [Fluentd](http://www.fluentd.org/)
+as structured log data. Then users can utilize any of the [various output plugins of Fluentd](http://www.fluentd.org/plugins)
+in order to write these logs to variety of destinations.
+
+In addition to the text of the log message itself, the `fluentd` log
+driver sends the following metadata in the structured log message:
+
+| Field            | Description |
+-------------------|-------------|
+| `container_id`   | The full 64-character container ID. |
+| `container_name` | The container name at the time it was started. If you use `docker rename` to rename a container, the new name is not reflected in the journal entries. |
+| `source`         | `stdout` or `stderr` |
+
+## Usage
+
+You can configure the default logging driver by passing the
+`--log-driver` option to the Docker daemon:
+
+    docker --log-driver=fluentd
+
+You can set the logging driver for a specific container by using the
+`--log-driver` option to `docker run`:
+
+    docker run --log-driver=fluentd ...
+
+## Options
+
+Users can specify some options for Fluentd logging driver by using `--log-opt`
+option, like `--log-opt NAME=VALUE`.
+
+### fluentd-address
+
+Fluentd logging driver connects to `localhost:24224` by default. Use
+`fluentd-address` option to specify another address to connect.
+
+    docker run --log-driver=fluentd --log-opt fluentd-address=myhost.local:24224 ...
+
+### fluentd-tag
+
+Every Fluentd's event has a tag, indicating where the log comes from. Fluentd logging
+driver uses `docker.{{.ID}}` as the tag by default.
+
+When specifying the value of this option, users can use the following markup tags:
+
+ - `{{.ID}}`: short container id (12 characters)
+ - `{{.FullID}}`: full container id
+ - `{{.Name}}`: container name
+
+## Note regarding container names
+
+The value logged in the `container_name` field (and `{{.Name}}` in tag)
+is the container name that was set at startup. If you use `docker rename`
+to rename a container, the new name will not be reflected in the fluentd messages.
+Fluentd messages will continue to use the original name.
+
+## Fluentd daemon management with Docker
+
+About `Fluentd` itself, see [the project webpage](http://www.fluentd.org)
+and [its documents](http://docs.fluentd.org/).
+
+To start using this logging driver, Fluentd daemon should be launched at somewhere
```

Comment:
Suggested rewrite for the first sentence. Also I want to know what will happen when the Fluentd daemon has not started yet.

Before using this logging driver, you need to launch a Fluentd daemon at `localhost:24224` or at another address that you specify with the `fluentd-address` option. If the logging driver fails to connect to the Fluentd daemon, [describe what will happen].